test/images/agnhost/Dockerfile_windows

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM BASEIMAGE
```

Comment:
instead of string/replace could we use ARG?

Reply:
Indeed, I prefered ARG instead of string/replace mechanism we have here. (See: https://github.com/kubernetes-sigs/windows-testing/blob/master/images/test-webserver/Dockerfile#L15). If people want it, I can do this, it'll make the Dockerfiles usable outside of the ``image-util.sh`` script.